src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -992,7 +992,7 @@ int Terminal::ViewStartIndex() const noexcept
 
 int Terminal::ViewEndIndex() const noexcept
 {
-    return _inAltBuffer() ? _altBufferSize.height : _mutableViewport.BottomInclusive();
+    return _inAltBuffer() ? _altBufferSize.height - 1 : _mutableViewport.BottomInclusive();

```

Comment:
@zadjii-msft are there any alt buffer conformance scenarios you'd like us to check before committing to this change?

Reply:
This I checked myself before I left last week, and seemed like it was just a typo tbh. Like, there's no reason to not have the -1